File: Drills/1_NUS_Kattis/2_JudgingMoose/JudgingMoose.java
import java.util.*;

public class JudgingMoose {
    public static void main(String[] args) throws Exception {
        Scanner sc = new Scanner(System.in);
        int left = sc.nextInt();
        int right = sc.nextInt();
        int result;
        
        if (left == 0 && right == 0){
            System.out.println("Not a moose");
        }
        
        if (left > right){
            result = left * 2;
            System.out.println("Odd "+result);
        }
            
        if (left < right){
            result = right * 2;
            System.out.println("Odd "+result);
        }
        
        if (left == right && left > 0){
            result = left * 2;
            System.out.println("Even "+result);
        }
        
    }
}

File: Drills/1_NUS_Kattis/2_JudgingMoose/JudgingMoose.c
#include <stdio.h>
int main(void){
    
    int result=0, left=0, right=0;
    
    scanf ("%d %d", &left, &right);
    
    if (left == 0 && right == 0) {
        printf("Not a moose");
    }
    
    if (left > right){
        result = left * 2;
        printf("Odd %d",result);
    }
            
    if (left < right){
        result = right * 2;
        printf("Odd %d",result);
    }
        
    if (left == right && left > 0){
        result = left * 2;
        printf("Even %d ",result);
    }
 
}